packages/terra-switch/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "terra-switch",
+  "main": "lib/Switch.js",
+  "version": "0.0.0",
+  "description": "The terra-switch component provides users a way to toggle between state ON and OFF.",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/terra-core.git"
+  },
+  "keywords": [
+    "switch",
+    "Cerner",
+    "Terra",
+    "terra-switch",
+    "UI"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/terra-core/issues"
+  },
+  "homepage": "https://github.com/cerner/terra-core#readme",
+  "scripts": {
+    "compile": "babel --root-mode upward src --out-dir lib --copy-files",
+    "lint": "npm run lint:js && npm run lint:scss",
+    "lint:js": "eslint --ext .js,.jsx . --ignore-path ../../.eslintignore",
+    "lint:scss": "stylelint src/**/*.scss",
+    "precompile": "rm -rf lib",
+    "test": "npm run jest && npm run wdio",
+    "jest": "jest --config ../../jestconfig.js",
+    "wdio-default": "cd ../.. && npx wdio ./packages/$npm_package_name/wdio.conf.js",
+    "wdio-lowlight": "cd ../.. && THEME=clinical-lowlight-theme npx wdio ./packages/$npm_package_name/clinical-lowlight-wdio.conf.js",
+    "wdio-fusion": "cd ../.. && THEME=orion-fusion-theme npx wdio ./packages/$npm_package_name/wdio.conf.js",
+    "wdio": "npm run wdio-default && npm run wdio-fusion && npm run wdio-lowlight"
+  },
+  "peerDependencies": {
+    "react": "^16.8.5",
+    "react-dom": "^16.8.5"
+  },
+  "dependencies": {
+    "classnames": "^2.2.5",
+    "keycode-js": "^2.0.1",
+    "prop-types": "^15.5.8",
+    "lodash.uniqueid": "^4.0.1"
```

Comment:
Do we need this?
Because i don't see it being used in the package

Reply:
Not required now, it was previously added for generating ID for switch button internally.
Removed from package.json : 48fce2d9ca73773ac6f9891feaa7f2a56b36f864